hardhat.config.ts

```diff
@@ -103,7 +103,7 @@ const config: HardhatUserConfig = {
       timeout: 12000000,
     },
     bscmainnet: {
-      url: "http://127.0.0.1:1248",
+      url: process.env.BSC_ARCHIVE_NODE_URL || "https://bsc-dataseed.binance.org/",
```

Comment:
I would keep `http://127.0.0.1:1248`, to force us to use Frame in the deployments to mainnet